Formula/libbitcoin-database.rb

```diff
@@ -0,0 +1,41 @@
+class LibbitcoinDatabase < Formula
+  desc "Bitcoin High Performance Blockchain Database"
+  homepage "https://github.com/libbitcoin/libbitcoin-database"
+  url "https://github.com/libbitcoin/libbitcoin-database/archive/v3.3.0.tar.gz"
+  sha256 "b4d98199ac4629a9857c1eb8819fe8166525bf2dca9ed790a9bbe5dc9c9e9186"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libbitcoin"
+
+  def install
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin"].opt_libexec/"lib/pkgconfig"
+
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
```

Comment:
This needs to use the new style `<<~EOS` instead of `<<-EOS.undent`

Reply:
👍 

Stupid question: Is the new style a new Ruby convention, or a Homebrew convention?

Reply:
The new style is a new-ish ruby feature https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

It only just now became possible in Homebrew because we moved to Ruby 2.3 to match the system version on High Sierra.